server/chat-plugins/usersearch.ts

```diff
@@ -0,0 +1,96 @@
+import {Utils} from '../../lib';
+
+const ONLINE_SYMBOL = ` \u25C9 `;
+const OFFLINE_SYMBOL = ` \u25CC `;
```

Comment:
These are used in several places; I wonder if they should be attached to the `Users` or `Chat` globals (maybe in a separate PR).